src/sidebar/components/sidebar-content.js

```diff
@@ -27,13 +27,11 @@ function SidebarContentController(
     self.rootThread = thread();
     self.selectedTab = state.selectedTab;
 
-    const counts = tabs.counts(state.annotations);
-
     Object.assign(self, {
-      totalNotes: counts.notes,
-      totalAnnotations: counts.annotations,
-      totalOrphans: counts.orphans,
-      waitingToAnchorAnnotations: counts.anchoring > 0,
+      totalNotes: store.countNotes(),
```

Comment:
@hmstepanek this gets removed once we rebase again and get new selection-tabs component right?

Reply:
yes-I'm gonna do that now once I rebase.